.github/workflows/vendor-updates.yml

```
@@ -0,0 +1,103 @@
+              --search "vendor: $name is behind upstream in:title" \
+              --json number --jq '.[0].number')
+
+            if [ -n "$existing" ]; then
+              gh issue comment "$existing" --repo "$REPO" --body \
+                "Still behind as of $(date -u +%Y-%m-%d): pinned \`$pinned\`, upstream \`$latest\`. ([run]($RUN_URL))"
+              gh issue edit "$existing" --repo "$REPO" --title "$title"
+              continue
+            fi
+
+            gh issue create --repo "$REPO" --title "$title" --body "$(cat <<EOF
+\`$name\` is vendored at \`$pinned\`; upstream's latest release is \`$latest\`.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)